scripts/utils.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const path = require('path');
+
+const BASE_DIR = 'test/';
+
+function collectTestFolders(strategy) {
+    const testFolder = path.resolve(path.join(process.cwd(), BASE_DIR));
+
+    return extractFolder(testFolder, [], strategy);
+}
+
+function extractFolder(folderToRead, folders = [], folderStrategy) {
```

Comment:
`extractFolder()` is supposed to be invoked via `collectTestFolders()`. It would be better to have a default export - `collectTestFolders()` (`extracFolder()` is not consumed elsewhere).